src/docker-images/azure-blob-adapter/wsgi.py

```diff
@@ -24,37 +24,51 @@ def application(request):
     :param PlainRequest request:
     '''
     try:
-        blob_name = request.headers['X-Tag']
-    except KeyError:
-        logger.exception('Key Error')
-        return Response(status=400)
+        if request.method == 'GET' and request.path == '/healthz':
+            def get_container():
```

Comment:
should define this function in top level. This actually define this method everytime liveness probe calls

Reply:
since logs are removed, inlined these function calls